yukyungdeok/2439.py

```diff
@@ -0,0 +1,5 @@
+# 2439번 별 찍기 - 2
+n = int(input())
+
+for i in range(1,(n+1)):
+    print(" " * (n-i) + "*" * i)
```

Comment:
```suggestion
    print(" " * (n-i) + "*" * i)
```
이건 ㄹㅇ 생각 못했는데 깜짝 놀랐네요 능숙한 print문 사용법....